packages/inscription-view/src/components/widgets/tag/Tags.tsx

```diff
@@ -5,16 +5,13 @@ import { IvyIcons } from '@axonivy/ui-icons';
 import { useReadonly } from '@axonivy/ui-components';
 import { useCombobox } from 'downshift';
 import { useKeyboard } from 'react-aria';
-import { useEditorContext } from '../../../context/useEditorContext';
-import { useMeta } from '../../../context/useMeta';
+
 import { useTranslation } from 'react-i18next';
 
-const Tags = (props: { tags: string[]; onChange: (tags: string[]) => void }) => {
+const Tags = (props: { tags: string[]; availableTags: string[]; onChange: (tags: string[]) => void; customValues: boolean }) => {
```

Comment:
availableTags should not be a string[]. I would introduce an interface `Tag` with `value` and `label` this is needed to display roles.

Reply:
maybe reuse SelectItem type

Reply:
ups sorry not needed. displayName was not shown on in the old dropdown. only the name of the role.